runtime/onert/core/include/backend/basic/train/TrainableBackendContextHelpers.h

```diff
@@ -43,9 +43,7 @@ ITensorRegistry *genTensors(backend::train::TrainableBackendContext &ctx,
       return;
     // NOTE Assuming there is no layout changes (Always assume NHWC or UNKNOWN)
     assert(tgraph.layout() != ir::Layout::NCHW);
-    ir::OperandInfo backend_info{obj.shape(), obj.typeInfo(), obj.info().memAllocType(),
-                                 obj.isConstant()};
-    tensor_builder->registerTensorInfo(ind, backend_info, ir::Layout::NHWC);
+    tensor_builder->registerTensorInfo(ind, obj.info(), ir::Layout::NHWC);
```

Comment:
Why don't you apply this patch to `BackendContextHelpers.h` as well?

Reply:
Ah, I didn't know that. I found this location during implementing exporting circle. I will fix it also.